src/scriptdev2/scripts/kalimdor/desolace.cpp

```diff
@@ -678,6 +678,74 @@ bool QuestAccept_npc_rigger_gizelton(Player* pPlayer, Creature* pCreature, const
     return true;
 }
 
+enum
+{
+    SPELL_CURSE_OF_MAGRAMI = 18159,
+    SPELL_GHOST_SPAWN_IN   = 17321,
+    SPELL_BLUE_AURA        = 17327,
+    SPELL_GREEN_AURA       = 18951,
+
+    FACTION_HOSTILE        = 16,
+};
+
+struct npc_magrami_spectre : public ScriptedAI
+{
+    npc_magrami_spectre(Creature* pCreature) : ScriptedAI(pCreature) { Reset(); }
+
+    uint32 m_uiCurseTimer;
+
+    void Reset() override
+    {
+        m_uiCurseTimer = 4000;
+    }
+
+    void JustAIInit() override
```

Comment:
Well, my understanding is that if you don't specify any triggered flag in DoCastSpellIfCan(), it will use automatically TRIGGERED_NONE.
If this isn't the case now, then we need to fix it.